src/overlays/actors/ovl_End_Title/z_end_title.c

```diff
@@ -71,29 +72,30 @@ void EndTitle_Draw(Actor* thisx, GlobalContext* globalCtx) {
     }
 
     {
+        // Makes the screen sepia and draws the title cards
```

Comment:
I could be wrong but I don't see anything in here that makes the screen sepia.

Reply:
gDPSetEnvColor(gfxCtx->overlay.p++, 0xFF, 0x78, 0x1E, 0x00);

this tint of orange is very much like the sepia that actually gets used. im not a gfx guy so i cant say if theres anything else but this is what i was going by

Reply:
I'm psure that's the outline for the title card textures, but I could be wrong.

Reply:
oh nvm the text itself has this orange background coloring

Reply:
also make sure to remove data and reloc .s files